keyboards/waldo/readme.md

```diff
@@ -0,0 +1,13 @@
+Waldo
+===
+
+A 60% Keyboard for use in the Holz case by Greenwald Designs
+
+Keyboard Maintainer: TODO
```

Comment:
Line 6 also will need two spaces at the end for GitHub to render this file as intended, once you have it filled out.

```suggestion
Keyboard Maintainer: TODO  
```